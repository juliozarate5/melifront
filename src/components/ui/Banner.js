import React from 'react'

export default function Banner() {
    return (
        <div className="row">
        <div className="col-lg-12">
            <a href="https://www.mercadolibre.com.co/" target="_blank">
                <img className="img-banner"  src="https://http2.mlstatic.com/D_NQ_NP_887755-MLA47696043671_092021-OO.webp" />
            </a>
        </div>
    </div>
    )
}
