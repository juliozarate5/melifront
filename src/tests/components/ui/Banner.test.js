import React from "react";
import { shallow } from "enzyme";
import Banner from "../../../components/ui/Banner";

describe('Banner tests', () => {

    const wrapper = shallow(<Banner />);

    test('showing component successful', () => {

        expect(wrapper).toMatchSnapshot()

    })
})