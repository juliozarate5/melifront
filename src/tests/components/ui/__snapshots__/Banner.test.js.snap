// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner tests showing component successful 1`] = `
<div
  className="row"
>
  <div
    className="col-lg-12"
  >
    <a
      href="https://www.mercadolibre.com.co/"
      target="_blank"
    >
      <img
        className="img-banner"
        src="https://http2.mlstatic.com/D_NQ_NP_887755-MLA47696043671_092021-OO.webp"
      />
    </a>
  </div>
</div>
`;
